topics/event-classes.md

```diff
@@ -6,32 +6,67 @@ body_class: event-classes
 | `list.view` &#10033; <span class="deprecated">DEPRECATED</span> | A list was viewed |  [<i class="yes fas fa-check-circle"><span class="sr-only">Available</span></i>](timeseries.html#list-view)  |  [<i class="yes fas fa-check-circle"><span class="sr-only">Available</span></i>](raw.html#list-view)  |
 | `list.item.click` &#10033; | An item in a list interacted with in some way  |  [<i class="yes fas fa-check-circle"><span class="sr-only">Available</span></i>](timeseries.html#list-item-click)  |  <i class="no fas fa-times-circle"><span class="sr-only">Not available</span></i>  |
 | `list.item.view` &#10033; <span class="deprecated">DEPRECATED</span> | An item in a list was expanded/navigated to  |  [<i class="yes fas fa-check-circle"><span class="sr-only">Available</span></i>](timeseries.html#list-item-view)  |  [<i class="yes fas fa-check-circle"><span class="sr-only">Available</span></i>](raw.html#list-item-view)  |
 | `list.item.external_link.click` &#10033; | The user exited the list to an external content location  |  [<i class="yes fas fa-check-circle"><span class="sr-only">Available</span></i>](timeseries.html#list-item-external_link-click)  |  [<i class="yes fas fa-check-circle"><span class="sr-only">Available</span></i>](raw.html#list-item-external_link-click)  |
 | `lti.link.followed` | An LTI launch to a reading list was invoked |  [<i class="yes fas fa-check-circle"><span class="sr-only">Available</span></i>](timeseries.html#lti-link-followed)  |  [<i class="yes fas fa-check-circle"><span class="sr-only">Available</span></i>](raw.html#lti-link-followed)  |
+| `resource.edit` | A resource was updated by a user |  [<i class="yes fas fa-check-circle"><span class="sr-only">Available</span></i>](timeseries.html#resource-edit)  | <i class="no fas fa-times-circle"><span class="sr-only">Not available</span></i> |
+
+&#10033; These events are only available when using [New List View](https://support.talis.com/hc/en-us/articles/360000008189).
+
+## Copyright Clearance
+
+| Class | Description | Timeseries data? | Raw data? |
+| :- | :- | :- | :- |
+|`digitisation.view`| A digitisation was viewed | [<i class="yes fas fa-check-circle"><span class="sr-only">Available</span></i>](timeseries.html#digitisation-view)  |  <i class="no fas fa-times-circle"><span class="sr-only">Not available</span></i>  |
+|`digitisation.print.showModal`| A digitisation print modal was opened | [<i class="yes fas fa-check-circle"><span class="sr-only">Available</span></i>](timeseries.html#digitisation-print-showmodal)  |  <i class="no fas fa-times-circle"><span class="sr-only">Not available</span></i>  |
+|`digitisation.print`| A digitisation was downloaded for printing | [<i class="yes fas fa-check-circle"><span class="sr-only">Available</span></i>](timeseries.html#digitisation-print)  |  <i class="no fas fa-times-circle"><span class="sr-only">Not available</span></i>  |
```

Comment:
This is only recorded in Elevate - it is when the user clicks "Print" there, for a Copyright Cleared resource. This is in addition to `player.print`

Reply:
In TADC, when the user downloads a digitisation from the print modal, we just record that as a `digitisation.download`. 

It is confusing, I know

Reply:
Hmm.. that is confusing.

Reply:
Actually you mention player.print. but we don't have an entry for player.print at all.

Reply:
in fact player.print doesn't appear in the census event config at all. Should it?

Reply:
I'm not really sure - who made the decision on which Elevate events we send through?